tiphub-io/tiphub
# Tip success and error emails

Closes #3 

### What This Does

Adds some infrastructure around sending emails via SendGrid (too lazy to setup my own email server) and adds emails for when a tip succeeds and fails.

Still a WIP since neither are rate limited right now, so it might get a little annoying. The error messages could also be much more descriptive.

### Steps to Test

1. Setup your .env with the new sendgrid credentials
1. Setup your node with a valid connection
2. Send yourself a tip, confirm you get an email
3. Turn off your node or somehow invalidate your connection settings
4. Send yourself a tip, confirm it fails and you get an email

## Screenshots

<img width="933" alt="Screen Shot 2019-04-30 at 5 45 23 PM" src="https://user-images.githubusercontent.com/649992/56995437-c7d9ea80-6b6f-11e9-923e-0695a04405a8.png">

<img width="944" alt="Screen Shot 2019-04-30 at 5 45 08 PM" src="https://user-images.githubusercontent.com/649992/56995441-c9a3ae00-6b6f-11e9-80b5-9e4c899895af.png">
